app/proc/telegram.go

```diff
@@ -122,32 +122,35 @@ func (client TelegramClient) tagLinkOnlySupport(htmlText string) string {
 }
 
 // getMessageHTML generates HTML message from provided feed.Item
-func (client TelegramClient) getMessageHTML(item feed.Item, withMp3Link bool) string {
-	description := string(item.Description)
-
-	description = strings.TrimPrefix(description, "<![CDATA[")
-	description = strings.TrimSuffix(description, "]]>")
-
-	// apparently bluemonday doesn't remove escaped HTML tags
-	description = client.tagLinkOnlySupport(html.UnescapeString(description))
-	description = strings.TrimSpace(description)
-	messageHTML := description
-
+func (client TelegramClient) getMessageHTML(item feed.Item, withMp3Link, trimCaption bool) string {
+	var header, footer string
 	title := strings.TrimSpace(item.Title)
 	if title != "" {
```

Comment:
this if/switch construct is a little bit award. Isn't all of this just a couple of ifs? i.e. `title != "" && item.Link == ""` and  `title != "" && item.Link != ""`